website/docs/d/networkfirewall_firewall_policy.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+subcategory: "Network Firewall"
+layout: "aws"
+page_title: "AWS: aws_networkfirewall_firewall_policy"
+description: |-
+  Retrieve information about a firewall policy.
+---
+
+# Data Source:  aws_networkfirewall_firewall_policy
+
+Retrieve information about a firewall policy.
+
+## Example Usage
```

Comment:
Can we provide examples for just `arn` and `name`. We can use Header 3 (`###`) to split them up. It will also be good to advise readers on the use cases for `arn` , `name`, and both of them together